cluster/src/main/java/org/apache/iotdb/cluster/coordinator/Coordinator.java

```diff
@@ -334,7 +345,8 @@ private TSStatus forwardPlan(List<PartitionGroup> partitionGroups, PhysicalPlan
   private TSStatus forwardPlan(Map<PhysicalPlan, PartitionGroup> planGroupMap, PhysicalPlan plan) {
     // the error codes from the groups that cannot execute the plan
     TSStatus status;
-    if (planGroupMap.size() == 1) {
+    // need to create substatus for multiPlan
+    if (planGroupMap.size() == 1 && !(plan instanceof CreateMultiTimeSeriesPlan)) {
```

Comment:
I think these specializations make the code difficult to maintain, so why not do permission checking in `TSServiceImpl`?

Reply:
TSServiceImpl did privilege check and mark it in plan.getResults() map, so server mode privilege works fine. 
But in cluster mode, if current user has no privilege and cannot create any timeseries, in Coordinator planGroupMap is always empty, then the processPartitionedPlan() method will always call upon itself to try split the plan. its an endless loop

so i think we need a check here to break out the loop when planGroupMap is empty.
please let me know if you have any workaround

Reply:
Still not quite clear of here.
Why does this branch not create substatus for multiPlans? And how about other multiPlans other than creating timeseries?

Reply:
in processPartitionedPlan method, it checks if (planGroupMap == null || planGroupMap.isEmpty()) first, when user has no privilege, the planGroupMap is empty, so processPartitionedPlan method calls on itself directly. It skips calling on forwardPlan(). thus cannot create substatus.